### ISSUE TITLE
Valid JSON not sending properly until formatted

### ISSUE THREAD

**Issue by bdavidhicks** [a85/POSTMan-Chrome-Extension#4925]
**Describe the bug**
Supplying valid JSON in request body does not send properly until request body is formatted using Ctrl+B shortcut.

**To Reproduce**
1. Use valid JSON that is **NOT** formatted using Ctrl+B like the below:
```
{
	"table": "COMPINFO",
	"columns": "COMPID,COMP_DESC",
	"filter": {
		"logic":"and",
		"filters":[{
			"field":"comp_desc",
			"operator":"contains",
			"value" :"HVAC"
		}]
	},
	"sortBy": [{
		"field":"comp_desc",
		"dir":"asc"
	}],
	"pageNumber": 1,
	"pageSize": 5
}
```
2. Observe there are no errors with the JSON
3. Send the request, the API ignores parts of the request body (I gathered from trial and error)
4. Response is not what I expected
5. Go back to request body and press `Ctrl+B` and see the body format like so:
```
{
    "table": "COMPINFO",
    "columns": "COMPID,COMP_DESC",
    "filter": {
        "logic": "and",
        "filters": [
            {
                "field": "comp_desc",
                "operator": "contains",
                "value": "HVAC"
            }
        ]
    },
    "sortBy": [
        {
            "field": "comp_desc",
            "dir": "asc"
        }
    ],
    "pageNumber": 1,
    "pageSize": 5
}
```
6. Send the request, the entire request body is accepted this time by the API
7. Receive a response that makes sense

**Expected behavior**
Sending JSON that is valid should not have to be formatted before sending request to receive the proper response. 

**Screenshots**
Before formatting:
![image](https://user-images.githubusercontent.com/17597693/43424528-ff099c02-940c-11e8-94b7-540b35cd72cf.png)

After formatting:
![image](https://user-images.githubusercontent.com/17597693/43424544-121447ac-940d-11e8-853d-55169499ce7d.png)

**App information from Settings > About:**
 - Postman for Windows
 - Version 6.2.2
 - win32 10.0.17134 / x64

**Comment by harryi3t** [comment]
@bdavidhicks Can you also post the screenshot of the error that you are getting?
Also, is the API endpoint that you are using public? If yes, please share the API details so that we can reproduce this on our end. If not, then please share the __exact__ raw JSON data with & without formatting.

**Comment by bdavidhicks** [comment]
@harryi3t @prashantagarwal I am not getting any error from the API and the API is not public. I already posted the exact JSON that I was sending both times above.

**Comment by prashantagarwal** [comment]
@bdavidhicks I just tried this but was unable to reproduce this. Can you try sending this with curl so that we can be sure that API server is not doing any sort of interpretations here.

**Comment by prashantagarwal** [comment]
@bdavidhicks Also in the unformatted JSON, do you have any newlines at the end of JSON ?

**Comment by bdavidhicks** [comment]
I'm on a Windows box at the moment and I was able to send the request using `Invoke-RestMethod` but I had to do a lot of conversion of the raw JSON in Powershell. I was able to recieve a response and it made sense for what I was sending. I looked closer at the raw JSON of the first request and I think the problem lies here:
```
			"value" :"HVAC"
```
as opposed to:
```
                "value": "HVAC"
```
The colon has a space before it in the first request but after formatting it places the space after the colon. I am not sure that I can replicate this using `Invoke-RestMethod` because of the conversion process of the raw object. Does that help?
